package.nls.json

```diff
@@ -62,11 +62,11 @@
     "jupyter.command.jupyter.runLinting.title": "Run Linting",
     "jupyter.command.jupyter.runFileInteractive.title": "Run Current File in Interactive Window",
     "jupyter.command.jupyter.debugFileInteractive.title": "Debug Current File in Interactive Window",
-    "jupyter.command.jupyter.runallcells.title": "Run All Cells",
+    "jupyter.command.jupyter.runAllCells.title": "Run All Cells",
```

Comment:
Is camel case required? Wondering why these changed.

Reply:
I'll unchanged them, it just seemed like a good improvement to me. But it's not necessary.

Reply:
I'll make an issue to make these consistent!

Reply:
Issue: https://github.com/microsoft/vscode-jupyter/issues/10381